packages/adapter-node/src/index.ts

```diff
@@ -0,0 +1,30 @@
+import 'cross-fetch/polyfill';
+import { INSTALLED } from '@pixi/core';
+import { ContextAttributes, ContextIds, IAdapter, settings } from '@pixi/settings';
+import gl from 'gl';
+import { NodeCanvasElement } from './NodeCanvasElement';
+import { NodeCanvasResource } from './NodeCanvasResource';
+import { loadNodeTexture } from './LoadNodeTexture';
+import { loadNodeFont } from './LoadNodeFont';
+import { loadJSON } from './temp/LoadJSON';
+import { loadSpritesheet } from './temp/LoadSpritesheet';
+
+INSTALLED.length = 0;
+INSTALLED.push(NodeCanvasResource);
+
+export const NodeAdapter = {
```

Comment:
I think adaptor is cool :D

I think we can simplify it to only a few functions! Might be best to discuss as I may be missing some details! This would result in less places where the Adaptor code needs to be used, and leverage the fact we have a pretty much a fully working canvas in there! 

createCanvas <-- keep
getContext <-- suggest we remove - this can be removed if we add that logic to the NodeCanvasElement (`canvas. getContext`)
getWebGLRenderingContext <--  suggest we remove, as we only use this in the `SCISSOR_TEST` checks . Lets modify the `ScissorSystem` class to to not store the glCOnst and instead access it using `gl.SCISSOR_TEST. we can use:

```
  protected contextChange(gl: IRenderingContext): void
    {
        this.glConst = gl.SCISSOR_TEST;
    }
```
getNavigator <-- keep